package.json

```diff
@@ -33,5 +33,8 @@
     },
     "simple-git-hooks": {
         "pre-commit": "bun run format && bun run lint:fix"
+    },
+    "dependencies": {
+        "merkletreejs": "^0.3.11"
```

Comment:
Don't think we need it here. It should be in the relevant package's package.json. Make sure the bun.lockb changes that way only.